bin/hdfcoinc/pycbc_page_injtable

```diff
@@ -45,34 +73,42 @@ else:
             newsnr = pycbc.events.ranking.newsnr(snr, chisq)
             
             found_names += ["%s SNR" % ifo, "CHISQ", "NewSNR"]
-            found_cols += [snr, chisq, newsnr]
+            found_cols += [numpy.round(snr,2), numpy.round(chisq,2), numpy.round(newsnr,2)]
             found_formats += ['##.##', '##.##', '##.##']
-          
-dec_dist = numpy.maximum(inj['eff_dist_h'][:][idx], inj['eff_dist_l'][:][idx])
+
+eff_dist = {'eff_dist_l':'Eff Dist(L)','eff_dist_h':'Eff Dist(H)','eff_dist_v':'Eff Dist(V)',
+            'eff_dist_k':'Eff Dist(K)','eff_dist_i':'Eff Dist(I)'}
+keys = inj.keys()
```

Comment:
If you look at https://ligo.gravity.cf.ac.uk/~koustav.chandra/LSC/images/pycbc_page_injtable/two_detector_found.html, you can see that we are given a Virgo effective distance even though it isn't involved in the search. This will be solved by using the file attribute for ifo list

Reply:
The two detector does content effective distance column corresponding to the Virgo detector and hence it displayed it.